app/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonTest.kt

```diff
@@ -172,23 +169,18 @@ class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
   }
 
   private fun capture(
-    background: @Composable () -> Color = { Color.Unspecified },
-    content: () -> View
+    @ColorRes background: Int? = null,
+    content: () -> View,
   ) {
-    composed(
-      size = IntSize(160, 64),
-      tags = listOf(type, size),
-    ) {
-      Box(
-        Modifier
-          .fillMaxSize()
-          .background(background())
-          .padding(BpkSpacing.Md),
-        contentAlignment = Alignment.TopStart
-      ) {
-        AndroidView(factory = { content() })
-      }
+    val wrapper = FrameLayout(testContext).apply {
```

Comment:
stopped relying on compose here. compose tests are much slower to run, so this should boost tests a little